Added changelog for 3.21.3 (#2716)
cfengine/masterfiles: CHANGELOG.md

3.21.3:

>     - Fixed 'regline' error messages which appear on systems (ex. CentOS 6) where /sys/hypervisor/uuid is not readable (ENT-9931)

Reviewer: ```suggestion
    - Fixed 'regline' error messages which appear on systems (ex. CentOS 6)
      where /sys/hypervisor/uuid is not readable (ENT-9931)
```

3.21.3:

>     - Guard against /sys/hypervisor/uuid not being readable (ENT-9931)

Reviewer: @craigcomstock I don't think your changelog entry conveys what was changed (for an end user). Here's my suggestion:

```suggestion
    - Fixed 'regline' error messages which appear on systems (ex. CentOS 6) where /sys/hypervisor/uuid is not readable (ENT-9931)
```

>     - Fixed lib/testing.cf tap output (CFE-4245)

Reviewer: Test only change, should not be in changelog:
```suggestion
```

>     - Fixed lib/testing.cf tap output (CFE-4245)
>     - Removed jq dependency in lib/testing.cf and fixed tap output (CFE-4242)

Reviewer 2: ```suggestion
    - Removed jq dependency in lib/testing.cf and fixed tap output (CFE-4242, CFE-4245)
```

We can merge these together instead of repeating how tap output was fixed.
